### ISSUE TITLE
Masthead - updates to support better logo layout

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#10788]
Placeholder issue for https://github.com/patternfly/patternfly/pull/6870

So far this update will just be to add a new container around the masthead toggle and logo, and rename a couple of containers within it.

**Comment by srambach** [comment]
We've added a wrapper around the toggle and logo in the masthead. One wrinkle is that 
- the name `masthead__main` was repurposed as the wrapper
- what was `masthead__main` is now `masthead__brand`
- what was `masthead__brand` is now `masthead__logo`

Old structure of the masthead was:
```
masthead
  masthead__toggle
  masthead__main
    masthead__brand
  masthead__content
```
New structure of the masthead is:
```
masthead
  masthead__main
    masthead__toggle
    masthead__brand
      masthead__logo
  masthead__content
```

**Comment by tlabaj** [comment]
closed by #10809
